idseq_dag/util/command.py

```diff
@@ -176,8 +176,11 @@ def wrapper(*args, **kwargs):
         def subprocess_scope(*args, **kwargs):
             with log.log_context("subprocess_scope", {"target": target.__qualname__, "original_caller": original_caller}):
                 target(*args, **kwargs)
-        p = multiprocessing.Process(target=subprocess_scope, args=args, kwargs=kwargs)
-        p.start()
+
+        with log.print_lock:  # important Voodoo magic to reduce deadlock dangers, remove at your peril
```

Comment:
Would it be possible to add a bit of additional explanation here? Something about `deadlock from CPython file handling` perhaps.

Reply:
Good idea, I'll try to elaborate in the comment.